src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/ScheduleController.kt

```diff
@@ -147,6 +148,13 @@ class ScheduleController(
     return scheduleService.getSdsExclusionLists()
   }
 
+  @Cacheable(TORERA_OFFENCE_CODES)
```

Comment:
I think the use case from CRD is to to pass in a list of offence codes and we return a list of offence codes that are TORRERA (if any) - so the logic is on our side  and CRD has to do the bare minimum 

Maybe even a true or false return would suffice,  but I think returning the subset of offence-code is probably better, in case they want to log or diagnose problems